### ISSUE TITLE
Publish data to channel by socketCluster object

### ISSUE THREAD

**Issue by fanweixiao** [TopCloud/socketcluster#37]
after `var sc = new SocketCluster(opts);`, I want to use `sc` object to publish data to channel, but can not find a way.

while I can use `worker` or `socket` to complete this job, but I want to use redis pub/sub to trigger sending data action, I don't want to do subscribing to redis on worker because I have to sub only once. So the `sc` creation is the best place to do redis pub/sub.

**Comment by jondubois** [comment]
I'm closing this issue. The optimal solution will be the Redis adapter - I opened a separate issue for it here: https://github.com/TopCloud/socketcluster/issues/38

In the meantime, the best way to interact with SC from external systems is by passing data to workers. You can use IDs to uniquely identify them.

**Comment by jondubois** [comment]
@slidenerd 

>> if I do worker.exchange.publish() from worker 0 and if a client is connected to worker 1 will they still get this data?

Yes, channels in SC are global so they are shared between all processes. If you use SCC, channels are also shared between all machines so it doesn't matter what process or machine the client is connected to.
